### PR TITLE
update rules

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: lib/rules/web/admin_console/4.4/common_ui_elements.xyaml [hunk 1]
```diff
@@ -597,10 +597,6 @@ check_restricted_access_text:
   params:
     content: Restricted Access
   action: check_page_contains
-check_no_configmaps_found_text:
-  params:
-    content: No Config Maps Found
-  action: check_page_contains
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/common_ui_elements.xyaml:603]: `check_no_configmaps_found_text` is no longer needed after cucushift#8345 is merged so removed it

File: lib/rules/web/admin_console/4.5/route.xyaml [hunk 2]
```diff
@@ -32,10 +32,6 @@ check_custom_dns_help_info:
   params:
     content: To use a custom route, you must update your DNS provider
   action: check_page_contains
-check_routes_web_page_contains:
-  element:
-    selector:
-      xpath: //*[contains(.,'<content>')]
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/route.xyaml:38]: this step is not needed since we updated the steps in cucushift#8346

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**